<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="okayPushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>561</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="popUpShutterTab">
    <attribute name="title">
     <string>shutterControl</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>323</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="shutterButtonHorizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="shutter1VerticalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="shutter1PushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>S1</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shutter1Label">
          <property name="text">
           <string>Shutter 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="shutter2VerticalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="shutter2PushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>S2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shutter2Label">
          <property name="text">
           <string>Shutter 2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="shutter3VerticalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="shutter3PushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>S3</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shutter3Label">
          <property name="text">
           <string>Shutter 3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="shutter4VerticalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="shutter4PushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>S4</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shutter4Label">
          <property name="text">
           <string>Shutter 4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="popUpTempTab">
    <attribute name="title">
     <string>tempControl</string>
    </attribute>
    <widget class="QTableWidget" name="popUpTempTableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>481</width>
       <height>251</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>4</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Control</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temperature</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ramp</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Cell1</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Cell2</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Cell3</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>Cell4</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="stepNameLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameInputLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Name</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="stepTimeDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Duration</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stepTimeUnitComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>seconds</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>minutes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hours</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Units</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
